### PR TITLE
Use appropriate zip fields for Cilium.Hubble

### PR DESCRIPTION
**Pull request by nebula-it**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Is there a linked Issue?https://github.com/microsoft/winget-create/issues/559#issuecomment-2463117810  https://github.com/microsoft/winget-pkgs/pull/189497

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/189500)

### CODE REVIEW

File: manifests/c/Cilium/Hubble/1.16.3/Cilium.Hubble.installer.yaml [hunk 1]
```diff
@@ -4,7 +4,11 @@
 PackageIdentifier: Cilium.Hubble
 PackageVersion: 1.16.3
 InstallerLocale: en-US
-InstallerType: portable
+nstallerType: zip
```
Comment (mdanish-kh) [inline manifests/c/Cilium/Hubble/1.16.3/Cilium.Hubble.installer.yaml:7]:
```suggestion
InstallerType: zip
```

**Review by mdanish-kh — CHANGES_REQUESTED** [review]

**Comment by nebula-it** [comment]
Hey @mdanish-kh, Any idea on the validation failure?

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Installation failed with exit code -1978335140
> 2024-11-07 12:39:04.278 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerCommonCore\Archive.cpp(33)\WindowsPackageManager.dll!00007FFED75C413B: (caller: 00007FFED753EBB5) ReturnHr(1) tid(d38) 80004005 Unspecified error
> 2024-11-07 12:39:04.280 [CLI ] Failed to extract archive with code -2147467259

Automated error analysis suggests -1978335140 may mean Failed to extract archive. Does the `InstallerUrl` actually end in `.zip` - or does it have a different extension? 

Automated error analysis suggests 80004005 may mean Resource missing

(Automated response - build 897.)

**Comment by nebula-it** [comment]
Installers have `.tar.gz` extension as you can see [here](https://github.com/microsoft/winget-pkgs/blob/master/manifests/c/Cilium/Hubble/1.16.3/Cilium.Hubble.installer.yaml#L12)

**Comment by stephengillie** [comment]
The package manager first downloads the file to a temporary location, with a temporary name, then renames the file and moves it to its target location. When `InstallerType: zip` is used, the existing file extension is clobbered and replaced with `.zip`. Then ZIP extraction fails, causing the exit code `-1978335140`. 

Usually this happens with `.exe` installers being renamed to `.zip`. But here, it appears to be renaming a `.tar.gz` file to `.zip`.  A workaround to get this working locally is to change the package manager setting `archiveExtractionMethod` from `shellApi` to `tar`. But this isn't currently supported in the pipelines:
- https://github.com/microsoft/winget-cli/issues/2899

**Comment by nebula-it** [comment]
Thank you for digging, I guess we just have to wait till it is implemented?
Or we can continue using the `InstallerType: portable` as it was used previously and undo changes in this PR.

**Comment by stephengillie** [comment]
Yes, we'll have to wait for the feature to be implemented before we can use it. 

There's nothing to undo, as this PR hasn't been merged yet. Feel free to close the PR if you do not want to make this change.

**Comment by nebula-it** [comment]
Hey @ItzLevvie, As noted [here](https://github.com/microsoft/winget-pkgs/pull/189500#issuecomment-2475244704) we are waiting on winget to implement this.

**Comment by pl4nty** [comment]
Can we remove [v1.16.3](https://github.com/microsoft/winget-pkgs/blob/master/manifests/c/Cilium/Hubble/1.16.3/Cilium.Hubble.installer.yaml) too? `InstallerType: portable` just renames the `.tar.gz` to `.exe`, so it can't actually be executed

**Comment by ItzLevvie** [comment]
[Policy] Blocking-Issur

**Comment by ItzLevvie** [comment]
[Policy] Blocking-Issue

**Pull request closed without merging**